app/server/lib/HostedStorageManager.ts

```diff
@@ -52,6 +52,11 @@ function checkValidDocId(docId: string): void {
   }
 }
 
+export interface HostedStorageCallbacks {
```

Comment:
Can you add a comment on what this interface is for.

Reply:
I added comments to the individual functions, as I was struggling to come up with a comment for this that wasn't just a paraphrased explanation of dependency injection. Open to suggestions on this!

Reply:
It would be good to add the comment though.

I don't understand why dbManager is not enough. What other things can change that behavior (with links).